src/app/containers/FrontPageSection/utilities/rowDetails.jsx

```diff
@@ -0,0 +1,47 @@
+import { flatten, isEmpty, isNil } from 'ramda';
+import { TopRow, LeadingRow, RegularRow } from '../../FrontPageStoryRows';
+
+// Returns the component that should be used for the first row.
+// This is one of TopRow, LeadingRow.
+const getFirstRowComponent = rowLength => {
+  switch (rowLength) {
+    case 1:
+      return { rowComponent: TopRow };
```

Comment:
Should these be `RowComponent` with a capital? Since it's a React component.

Reply:
I wasn't sure about the convention when it's a key in an object, but thinking about it, you're probably right and it'll make it clearer when looking at the object that this element is a React component.